src/Gateway.php

```diff
@@ -15,6 +15,15 @@ public function __construct ( $host, $request_options = array())
         $this->description = 'Endpoint SDK for using the gateway';
     }
 
+    public function login($credentials) {
+        if (!is_array($credentials))
+            throw new Exception('Login requires first param to be an Associative Array');
+
+        $resp =  $this->post('v1/gateway/login', $credentials);
+        $jwt = $resp->headers['authorization'];
+        $this->setJwt($jwt);
```

Comment:
Is this not done after every response?

Reply:
Since we're overriding the inherited method, we have to re-implement the `setJwt()` call.